### PR TITLE
Update emc-update-impl.bash

### PR DESCRIPTION
**Pull request by KdVos**
As far as I could find the map_server is not included in ros-noetic-desktop-full, thus we need to install it seperately/need to make sure it is installed

 #19

### CODE REVIEW

File: setup/emc-update-impl.bash [hunk 1]
```diff
@@ -118,7 +118,7 @@ if [ "$ROBOT_REAL" = true ] ; then
 fi
 
 # 3) Install dependencies
-_make_sure_installed ros-"$EMC_ROS_DISTRO"-cv-bridge ros-"$EMC_ROS_DISTRO"-image-geometry ros-"$EMC_ROS_DISTRO"-tf libassimp-dev ros-"$EMC_ROS_DISTRO"-message-runtime ros-"$EMC_ROS_DISTRO"-message-generation ros-"$EMC_ROS_DISTRO"-roscpp ros-"$EMC_ROS_DISTRO"-nav-msgs ros-"$EMC_ROS_DISTRO"-shape-msgs
+_make_sure_installed ros-"$EMC_ROS_DISTRO"-cv-bridge ros-"$EMC_ROS_DISTRO"-map-server ros-"$EMC_ROS_DISTRO"-image-geometry ros-"$EMC_ROS_DISTRO"-tf libassimp-dev ros-"$EMC_ROS_DISTRO"-message-runtime ros-"$EMC_ROS_DISTRO"-message-generation ros-"$EMC_ROS_DISTRO"-roscpp ros-"$EMC_ROS_DISTRO"-nav-msgs ros-"$EMC_ROS_DISTRO"-shape-msgs
```
Comment (MatthijsBurgh) [inline setup/emc-update-impl.bash:121]:
```suggestion
_make_sure_installed libassimp-dev ros-"${EMC_ROS_DISTRO}"-cv-bridge ros-"${EMC_ROS_DISTRO}"-image-geometry ros-"${EMC_ROS_DISTRO}"-map-server ros-"${EMC_ROS_DISTRO}"-message-generation ros-"${EMC_ROS_DISTRO}"-message-runtime ros-"${EMC_ROS_DISTRO}"-nav-msgs ros-"${EMC_ROS_DISTRO}"-roscpp ros-"${EMC_ROS_DISTRO}"-shape-msgs ros-"${EMC_ROS_DISTRO}"-tf
```
Update this one we replacing `tf` by `tf2`

**Review by MatthijsBurgh — CHANGES_REQUESTED** [review]

File: setup/emc-update-impl.bash (hunk 1 above)
Comment (KdVos) [reply setup/emc-update-impl.bash:121]:
```suggestion
_make_sure_installed libassimp-dev ros-"${EMC_ROS_DISTRO}"-cv-bridge ros-"${EMC_ROS_DISTRO}"-image-geometry ros-"${EMC_ROS_DISTRO}"-map-server ros-"${EMC_ROS_DISTRO}"-message-generation ros-"${EMC_ROS_DISTRO}"-message-runtime ros-"${EMC_ROS_DISTRO}"-nav-msgs ros-"${EMC_ROS_DISTRO}"-roscpp ros-"${EMC_ROS_DISTRO}"-shape-msgs ros-"${EMC_ROS_DISTRO}"-tf2
```

To be sure, _ros-"${EMC_ROS_DISTRO}"-tf_ needs to be changed to _ros-"${EMC_ROS_DISTRO}"-tf2_ ?

Comment (MatthijsBurgh) [reply setup/emc-update-impl.bash:121]: last `tf`usage:
https://github.com/tue-robotics/emc_simulator/blob/c8c7d7eea1427095c03987f8f9df47f85f14334c/src/virtualbase.h#L139

Comment (KdVos) [reply setup/emc-update-impl.bash:121]: Removed

**Review by MatthijsBurgh — APPROVED** [review]

**Pull request merged**